### PR TITLE
Fix social volume aligning of datetimes

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary
Fix issues where the volumes are not summed properly if some of the sources return less or more data

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

**Comment by tspenov** [comment]
@IvanIvanoff could you add also a test

**Comment by IvanIvanoff** [comment]
@tspenov the tests mock the response from tech_indicators, so the logic passes through this code

**Comment by tspenov** [comment]
@IvanIvanoff ok, but is there anything testing exactly the issue in the description ?

File: lib/sanbase/social_data/social_volume.ex [hunk 1]
```diff
@@ -206,6 +188,21 @@ defmodule Sanbase.SocialData.SocialVolume do
     end)
   end
 
+  defp sum_by_datetime(data) do
```
Comment (tspenov) [inline lib/sanbase/social_data/social_volume.ex]: Can you extract this duplicated function

**Pull request merged**